### PR TITLE
issue #296

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

**Comment by jackzampolin** [comment]
https://github.com/influxdb/influxdb.com/issues/296

File: content/docs/v0.9/administration/logs.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ InfluxDB must be restarted to pick up any changes to `/etc/default/influxdb`.
 
 ## Using logrotate
 
-You can use [logrotate](http://manpages.ubuntu.com/manpages/hardy/man8/logrotate.8.html) to rotate the log files generated by InfluxDB. Be sure to enable `copytruncate` when rotating the log file, so InfluxDB can continue to write to the file after it has been rotated.
+You can use [logrotate](http://manpages.ubuntu.com/manpages/hardy/man8/logrotate.8.html) to rotate the log files generated by InfluxDB. If using the package install, the config file for logrotate is installed at `/etc/logrotate.d`.  You can view the file [here](https://github.com/influxdb/influxdb/blob/master/scripts/logrotate).
```
Comment (otoolep) [inline content/docs/v0.9/administration/logs.md]: Technically it is installed *in* that directory, not "at".

**Comment by otoolep** [comment]
+1

**Comment by jackzampolin** [comment]
@otoolep fixed

**Pull request merged**